app/models/gids_redis/gids.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require 'common/models/redis_store'
+require 'common/models/concerns/cache_aside'
```

Comment:
I don't think these are necessary once you fix the name/path of this class.

Reply:
removed